@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -0,0 +1,989 @@
+              <div class="radio-container">
+                <UiRadioButton
+                  v-model="vmState.installMode"
+                  :disabled="!vmState.new_vm_template"
+                  accent="brand"
+                  value="ISO"
+                >
+                  {{ $t('new-vm.iso-dvd') }}
+                </UiRadioButton>
+                <UiRadioButton
+                  v-model="vmState.installMode"
+                  :disabled="!vmState.new_vm_template"
+                  accent="brand"
+                  value="PXE"
+                >
+                  {{ $t('new-vm.pxe') }}
+                </UiRadioButton>
+              </div>
+              <FormSelect v-if="vmState.installMode === 'ISO'" v-model="vmState.selectedVdi">
+                <template v-for="(VDIs, srName) in VDIsGroupedBySrName" :key="srName">
+                  <optgroup :label="srName">
+                    <option v-for="vdi in VDIs" :key="vdi.$ref" :value="vdi.$ref">
+                      {{ vdi.name_label }}
+                    </option>
+                  </optgroup>
+                </template>
+              </FormSelect>
+            </div>
+          </div>
+          <!-- SYSTEM SECTION -->
+          <UiTitle>{{ $t('new-vm.system') }}</UiTitle>
+          <!-- <UiToggle v-model="vmState.toggle">{{ $t('new-vm.multi-creation') }}</UiToggle> -->
+          <div class="system-container">
+            <div class="col-left">
+              <UiInput v-model="vmState.vm_name" accent="brand" href="''">{{ $t('new-vm.vm-name') }}</UiInput>
+              <!-- <UiInput v-model="vmState.tags" :label-icon="faTags" accent="brand" href="''"> -->
+              <!-- {{ $t('new-vm.tags') }} -->
+              <!-- </UiInput> -->
+              <div>
+                <UiLabel accent="neutral" href="''">{{ $t('new-vm.boot-firmware') }}</UiLabel>
+                <FormSelect v-model="vmState.boot_firmware">
+                  <option v-for="boot in getBootFirmwares" :key="boot" :value="boot">
+                    {{ boot === undefined ? t('bios-default') : boot }}
+                  </option>
+                </FormSelect>
+              </div>
+              <div
+                v-if="vmState.boot_firmware === 'uefi' || templateHasBiosStrings"
+                v-tooltip="{
+                  placement: 'top-start',
+                  content:
+                    vmState.boot_firmware !== 'uefi'
+                      ? $t('new-vm.boot-firmware-bios')
+                      : $t('new-vm.boot-firmware-uefi'),
+                }"
+              >
+                <UiCheckbox v-model="getCopyHostBiosStrings" accent="brand" disabled>
+                  {{ $t('new-vm.copy-host') }}
+                </UiCheckbox>
+              </div>
+              <div v-else>
+                <UiCheckbox v-model="getCopyHostBiosStrings" accent="brand">
+                  {{ $t('new-vm.copy-host') }}
+                </UiCheckbox>
+              </div>
```

Comment:
```suggestion
<div
                v-tooltip="
                  vmState.boot_firmware === 'uefi' || templateHasBiosStrings
                    ? {
                        placement: 'top-start',
                        content:
                          vmState.boot_firmware !== 'uefi'
                            ? $t('new-vm.boot-firmware-bios')
                            : $t('new-vm.boot-firmware-uefi'),
                      }
                    : undefined
                "
              >
                <UiCheckbox
                  v-model="getCopyHostBiosStrings"
                  accent="brand"
                  :disabled="vmState.boot_firmware === 'uefi' || templateHasBiosStrings"
                >
                  {{ $t('new-vm.copy-host') }}
                </UiCheckbox>
              </div>
```
And make condition a _computed_